content/content/oop_macro.md

```diff
@@ -206,6 +209,11 @@ class Dog of Animal:
 class Cat of Animal:
```

Comment:
There is no constructor proc for the Animal class, so I don't see anything to update